src/Tests/Microsoft.NET.Publish.Tests/PublishItemsOutputGroupTests.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using FluentAssertions;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.Commands;
+using Microsoft.NET.TestFramework.ProjectConstruction;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.NET.Publish.Tests
+{
+    public class PublishItemsOutputGroupTests : SdkTest
+    {
+        public PublishItemsOutputGroupTests(ITestOutputHelper log) : base(log)
+        {
+        }
+
+        private readonly static List<string> FrameworkAssemblies = new List<string>()
+        {
+            "api-ms-win-core-console-l1-1-0.dll",
+            "System.Runtime.dll",
+            "WindowsBase.dll",
+        };
+
+        [Fact]
+        public void GroupPopulatedWithRid()
+        {
+            var testProject = this.SetupProject();
+            var testAsset = _testAssetsManager.CreateTestProject(testProject);
+
+            var restoreCommand = new RestoreCommand(Log, testAsset.Path, testProject.Name);
+            restoreCommand
+                .Execute()
+                .Should()
+                .Pass();
+
+            var buildCommand = new BuildCommand(Log, testAsset.Path, testProject.Name);
+            buildCommand
+                .Execute("/p:RuntimeIdentifier=win-x86", "/t:PublishItemsOutputGroup")
+                .Should()
+                .Pass();
+
+            var testOutputDir = new DirectoryInfo(Path.Combine(testAsset.Path, testProject.Name, "TestOutput"));
+            Log.WriteLine("Contents of PublishItemsOutputGroup dumped to '{0}'.", testOutputDir.FullName);
+
+            // Check for the existence of a few specific files that should be in the directory where the 
+            // contents of PublishItemsOutputGroup were dumped to make sure it's getting populated.
+            testOutputDir.Should().HaveFile($"{testProject.Name}{Constants.ExeSuffix}");
```

Comment:
My previous comment on this line was incorrect as I had originally meant it only for the one in `GroupNotPopulatedWithoutRid`.  This one should be explicitly `.exe` rather than using `Constnats.ExeSuffix` since we're building for Windows.  Sorry about the confusion.

Reply:
Got it, fixed